ark/utils/segmentation_utils_test.py

```diff
@@ -192,3 +193,37 @@ def test_transform_expression_matrix_multiple_compartments():
     for cell in cell_data.cell_id:
         arcsinh_vals = np.arcsinh(cell_data.loc[:, cell, modified_cols].values)
         assert np.array_equal(arcsinh_data.loc[:, cell, modified_cols].values, arcsinh_vals)
+
+
+def test_visualize_segmentation():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        channel_xr = test_utils.make_images_xarray(np.zeros((2, 50, 50, 5)))
+        overlay_channels = channel_xr.channels.values[:2],
```

Comment:
I think you're trying to address Noah's earlier comment about different chan lists, but you're still creating two channels; having 5 fovs doesn't fix it.  Instead:
```
channel_xr = test_utils.make_images.xarray(np.zeros((3, 50, 50, 1)))
overlay_channels = [channel_xr.channels.values[:2], channels_xr.channels.values[1:3]]
```
this should more realistically cover the case Noah addressed earlier.